### PR TITLE
chore(travis): Move Travis integration to igor-monitor-travis

### PR DESCRIPTION
**Pull request by jervi**
Had to move some classes to `igor-core` to be able to do this, as both Jenkins and Travis (maybe other ci's as well?) depends on them. We could probably consider moving some of this to kork instead.
Also converted a bunch of classes to Java. No Groovy left in `igor-core` (except tests) \o/

### CODE REVIEW

**Comment by jervi** [comment]
@robzienert Can you please take a look? 😃

**Comment by jervi** [comment]
@spinnaker/oss-reviewers PTAL

File: igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/TravisCache.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Netflix, Inc.
+ * Copyright 2020 Schibsted ASA.
```
Comment (robzienert) [inline igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/TravisCache.java:2]: Mind the copyright changes that alter companies.

Comment (jervi) [reply igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/TravisCache.java:2]: Oh, sorry. I did change a whole lot of them back, but seems this one slipped though the cracks. Bad IntelliJ!
I’ll fix it in the morning.

**Comment by jervi** [comment]
> Are any of these things used by other services?
No, I don't think so. It used to be artifact related classes, but they're already extracted.

**Pull request merged**